trlx/model/nn/ppo_models.py

```diff
@@ -162,10 +163,12 @@ def loss(
             old_values - self.cliprange_value,
             old_values + self.cliprange_value,
         )
+        n = mask.sum()
```

Comment:
I'm not sure but shouldn't n be vector valued where each component is the size of the ith generation? (Before endoftext)

Reply:
it is used only in reduction to scalars so no